### ISSUE TITLE
how to control the arm speed in gazebo

### ISSUE THREAD

**Issue by richardsanqi** [lihuang3/ur5_notebook#23]
hey，lihuang！I just want to control the speed of the arm in gazebo by the jacobian matrix.i find you in blocks_spawner.cpp to control the red_box speed,do you have some examples just use code to control the speed of control in gazebo? I am in this trouble for a long time.

**Comment by lihuang3** [comment]
No, we did not test the speed control as it was not within the scope of this project. You may find solutions in universal robot communities/forums.

**Comment by richardsanqi** [comment]
thanks a lot  lihuang,and the patient solution before.I will have a try.
